@here/harp-datasource-protocol/lib/ThreeBufferUtils.ts

```diff
@@ -19,6 +18,17 @@ import {
     InterleavedBufferAttribute
 } from "./DecodedTile";
 
+type TypedArray =
```

Comment:
This is necessary, yes, but we could consider to deprecate this type and instead use ArrayLike<number>, because that is the type in the three-ts-types, see: https://github.com/three-types/three-ts-types/blob/master/types/three/src/core/BufferAttribute.d.ts

Reply:
What about ``ArrayBufferView``?

Reply:
Internally it uses ArrayLike<number> in the three-ts-types, but I guess that works also.